curiouslearning/FeedTheMonsterJS
# feat: Refactor the Gameplay UI from canvas to HTML, using multiple/dynamic backgrounds via BackgroundComponent (FM-295)

# Changes
- Update index.html and add new div called background
- Created background folder under components folder
- Created background html generator
- Created base background component

# How to test
- Open the start screen of FTM.
- Tap or click any part of the start screen to proceed to the next screen
- In the level selection screen, select a level and then play
- Make sure background is working properly
- Make sure background is changing dynamically depending in the level

Ref: [FM-295](https://curiouslearning.atlassian.net/browse/FM-295)


[FM-295]: https://curiouslearning.atlassian.net/browse/FM-295?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
